src/components/autocomplete/js/autocompleteController.js

```diff
@@ -601,6 +601,8 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
     if (!noBlur) {
       ctrl.hidden = shouldHide();
       evalAttr('ngBlur', { $event: $event });
+    } else {
```

Comment:
We should add
```
@param {Event=} $event
```
to the JSDoc to mark it as optional since this function is most often called without an argument.

This is also why one of the tests is failing with
```js
TypeError: Cannot read property 'stopImmediatePropagation' of undefined
    at MdAutocompleteCtrl.blur (src/components/autocomplete/js/autocompleteController.js:605:14)
```
and
```js
TypeError: $event is undefined
blur@src/components/autocomplete/js/autocompleteController.js:605:7
```